src/plugins/bfetch/server/streaming/create_compressed_stream.ts

```diff
@@ -50,6 +97,13 @@ export const createCompressedStream = <Response>(
       }),
       finalize(() => {
         output.end();
+
+        if (analyticsStart && metricCollector) {
+          reportPerformanceMetricEvent(
+            analyticsStart,
+            metricCollector.getEBTPerformanceMetricEvent()
+          );
+        }
```

Comment:
@thomasneirynck I did few local runs I don't see this new event reported. I get `saved_object_loaded_time`, then `dashboard_loaded`, but no `bfetch_server_encoding`

You give it a shot locally `node scripts/run_performance.js --journey-path x-pack/performance/journeys_e2e/ecommerce_dashboard.ts` and search for `event_type: 'performance_metric' in console logs

Reply:
oki, thx, will check ..

Reply:
@dmlemeshko thank you! 

😱 import should have been from core-plugin. See https://github.com/elastic/kibana/pull/179663/files#diff-c2f4315a6f36903134986650705305dccd2ed9f3887dd513afc65a6314221e75R111

Reply:
Still no luck to see `bfetch_server_encoding` event locally. Does it work on your end?

Reply:
confirmed https://github.com/elastic/kibana/pull/179663#pullrequestreview-1994299349